README.md

```diff
@@ -1,2 +1,116 @@
-# covid
-This application show covid 19 data from around the world based on M-Media-Group API
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid 19
+
+This Repo is for showing  Covid-19 Cases is is based on M-Media-Group API. It is developed using React and Redux, It is one of task in Microverse React-Redux Curricullum .
+
+## Built With
+
+Technologies
+
+🔷HTML
+
+🔷CSS
+
+🔷JAVASCRIPT
+
+🔷WEBPACK
+
+🔷React
+
+🔷Redux
+
+Tools
+
+💠VISUAL STUDIO CODE
+
+💠GITHUB
+
+💠LINTERS (Sylint, ESlint)
+
+# Project Name
+
+Covid 19
+
+# Live Demo
+ [Live Link](https://62e1afc66146f605f91417ee--covid-wsk.netlify.app/)
+<!-- [Live Link Netlify](https://62c71fa33074b5205add1b31--profound-pegasus-6e0c9a.netlify.app/) -->
+Comming Soon
+
+## Getting Started
+
+To get a local copy up and running follow these sample example steps.
+
+-A text editor(preferably Visual Studio Code)
+
+### Prerequisites
+
+-Need a browser to run and download the repo
+-Git is recommended for downloding the repo.
+
+### Install GIT
+
+You can install git from the following site.
+  -  [Git](https://git-scm.com/downloads)
+
+### Usage
+Free to use for your work
+
+#### Clone this repository
+
+```bash
+$ git clone https://github.com/waqaskanju/book-cms.git
+
+$ cd [project_name]
+
+```
+#### No restraction  on usage.
+
+### Run tests
+
+There is a test file (linters.yml) for this project inside the .github/workflows folder.
+
+It executes three types of tests inside github actions.
+
+1.lighthouse
+
+2.webhint
+
+3.stylint
+
+4.Eslint
+
+
+### Deployment
+
+It can run on any browser. 
+Mobile Screen size below < 786
+Desktop Screen Size above >786
+
+## Authors
+
+👤 **Waqas Ahmad**
+
+- GitHub: [@githubhandle](https://github.com/waqaskanju)
+- Twitter: [@twitterhandle](https://twitter.com/waqaskanju)
+- LinkedIn: [@LinkedIn](https://linkedin.com/in/waqaskanju)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This file is adopted from Microverse. Thanks for assome there work.
+- The css strucutre is taken from Cretive Common Website [Link](https://creativecommons.org/licenses/by-nc/4.0/)
```

Comment:
- [ ] the acknowledgement for the writer should be more specific like below for example:
Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

Reply:
Thanks. done .
![image](https://user-images.githubusercontent.com/13853450/181384960-badbea4d-5a6f-4cc8-85a4-f37f4cc4174d.png)